cmd/prometheus/main.go

```diff
@@ -391,6 +392,9 @@ func main() {
 	serverOnlyFlag(a, "rules.alert.resend-delay", "Minimum amount of time to wait before resending an alert to Alertmanager.").
 		Default("1m").SetValue(&cfg.resendDelay)
 
+	serverOnlyFlag(a, "rules.async-rule-eval", "Allow rules within a rule group to evaluate asynchronously if they have no dependencies.").
```

Comment:
Evaluate synchronously... the flag should be changed

Reply:
Resolving this one; I've made some changes so this isn't relevant anymore.